test/server/interactions/interactions-network-generation-test.js

```diff
@@ -150,32 +95,4 @@ describe('Interactions network to cy.js conversion', function(){
 
   });
 
-  it('filters nodes with 0 degree', function(){
```

Comment:
I still think it is useful to have a test showing that we filter degree 0 nodes.

Reply:
I agree it's better to catch the edgecases as best as possible with the tests